src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components /defaultIconInput.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {InputTypes} from '../types';
+import {AppIcon} from '../../../../types/iconsTypes/enums';
+import * as I from '../styles';
+
+const DefaultIconInput = () => (
```

Comment:
@divyanshu-rawat Can you review this component?

Reply:
@Marvin9 little busy, please feel free to review it & leave your thoughts :)

Reply:
@Meghana-12 please don't put space while creating directory, it causes problem in windows machine while cloning the repo. 
![Screenshot 2020-04-06 at 18 42 03](https://user-images.githubusercontent.com/13464678/78582900-52be6d00-7836-11ea-97c2-a3f23bab7778.png)